src/render.js

```diff
@@ -46,14 +36,14 @@ async function selectSource(source, display) {
 let currentDisplay = null
 let dispx = 0
 let dispy = 0
-ipcRenderer.on("update-screen-to-capture", async (event, display) => {
-  cons.log(`>> update display: ${display.id}`)
+ipcRenderer.on("update-screen-to-capture",  async (event, { display, sourceId }) => {
+  cons.log(`>> update display: ${display.id}, ${sourceId}`);
   if (!currentDisplay || display.id !== currentDisplay.id) {
```

Comment:
I think it would be nice if we moved this condition to `determineScreenToCapture()` since the query for available sources has moved there